apps/subscriber-app/src/app/app.tsx

```diff
@@ -17,7 +17,7 @@ const AppRouters = () => {
   return (
     <Router>
       <Routes>
-        <Route path="/subscriptions/:realm" element={<PrivateApp />} />
+        <Route path="/subscriptions/:realm/*" element={<PrivateApp />} />
```

Comment:
I'd like to confirm one thing is that are we going to fix some previous routing issue here? I am asking this question because when we add a wild card, I expect we will introduce some new routes within the privateApp. But, I have not seen any update there.

Reply:
Sorry, this is a leftover from a previous PR - Currently there is a flow that uses the phone to verify code that is currently broken in dev/uat because I'm burying the route by not doing this